src/input-capture.c

```diff
@@ -124,6 +112,36 @@ input_capture_session_new (GVariant         *options,
   return (InputCaptureSession*)session;
 }
 
+gboolean
+input_capture_session_can_request_clipboard (InputCaptureSession *session)
+{
+  if (session->clipboard_requested)
+    return FALSE;
+
+  if (impl_version < 2)
+    return FALSE;
+
+  switch (session->state)
```

Comment:
Why a switch case when we're checking just one condition?

`return session->state == INPUT_CAPTURE_SESSION_STATE_INIT`